### ISSUE TITLE
A11y: Missing Aria-Label in product-slider

### ISSUE THREAD

**Issue by mfg2025** [shopware/platform#6927]
*PHP Version:* 8.3   | *Shopware Version:* 6.6.10.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

Buttons (arrows) in product sliders do not have a name accessible to screen readers:
`<button class="base-slider-controls-prev product-slider-controls-prev is-arrow-prev-outsi…" aria-controls="tns1" tabindex="-1" data-controls="prev"> and <button class="base-slider-controls-next product-slider-controls-next is-arrow-next-outsi…" aria-controls="tns1" tabindex="-1" data-controls="next">`

*_Expected behaviour:_*

`<button class="base-slider-controls-prev product-slider-controls-prev is-arrow-prev-outsi…"  aria-controls="tns1"  tabindex="-1"  data-controls="prev" aria-label="previous product"></button>
 and 
<button class="base-slider-controls-prev product-slider-controls-prev is-arrow-prev-outsi…"  aria-controls="tns1"  tabindex="-1"  data-controls="prev" aria-label="next product"></button>
`

*_How to reproduce:_*

Check the shopware site with: https://pagespeed.web.dev or in the Code

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40727**



[created from NEXT-40727, comment 540516]

**Comment by mfg2025** [comment]
Thanks

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40727, comment 540590]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.7.0.0**

 This issue will be / was solved in Shopware version ***6.7.0.0***.



[created from NEXT-40727, comment 542610]
